ci/upload_to_owncloud.sh

```diff
@@ -13,10 +13,13 @@ rootdir=$(realpath "$scriptdir/../")
 . "$rootdir/tools/functions.sh"
 
 usage() {
-    echo "usage: $(basename "$0") [-hv] <user> <password> <remote-path> <local-path>"
+    echo "usage: $(basename "$0") [-hv] [--overwrite-remote-path <temp-path>] <user> <password> <remote-path> <local-path>"
     echo "  options:"
     echo "      -h|--help - display this help."
-    echo "      -v|--verbose - enable verbosity"
+    echo "      -v|--verbose - enable verbosity."
+    echo "      --overwrite-remote-path - replace any existing folder on remote-path with local-path."
```

Comment:
This option is unclear to me. I would expect this to be a true/false value, which means whether or not to overwrite files and folders in the remote if they already exist.
I understand why you would want to first upload to a temporary location - so that we won't get to a case where there is no prplmesh.ipk in owncloud, which is great, but I think you should use a hard coded temporary path and keep the option simple for the user.

Reply:
I agree with you the option is obscure, but we cannot use a hardcoded path as CI jobs run concurrently (you could end up having 2 pipelines uploading to the same folder at the same time...).

Reply:
We can generate a temporary name - using uuidgen, pwgen, `dd if=/dev/urandom bs=24 count=1 | sha256sum`, whatever. With 20 random bytes and a halfway decent RNG the chance of conflicts is about as large as that the universe is destroyed _right_this_instant_ :-)

Reply:
The idea behind a configurable temporary name was to be able to use the job ID in CI so that we can easily identify the jobs if needed, but it's not that useful for now so ok I'll use uuidgen.